src/app/rosetta/test-agent/agent.ml

```diff
@@ -512,17 +512,10 @@ let get_consensus_constants ~logger :
     Filename.concat home ".coda-config"
   in
   let config_file =
-    let mina_config_file = "MINA_CONFIG_FILE" in
-    let coda_config_file = "CODA_CONFIG_FILE" in
-    match Sys.getenv mina_config_file,Sys.getenv coda_config_file with
-    | Some config_file,_ ->
+    match Sys.getenv "MINA_CONFIG_FILE" with
+    | Some config_file ->
         config_file
-    | None,Some config_file ->
-      [%log warn] "Using deprecated environment variable %s, please use %s instead"
```

Comment:
Should users that still have their environment set up to only use the `CODA_CONFIG_FILE` variable get a nonsupport error instead?

Reply:
Well, that's a choice to be made. I thought it better to have a transitional period where there's a warning, and still allow the old variable to be used.

Reply:
Yup, that makes sense to me. I am approving it 👍